src/pages/Landing.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import Footer from '../components/Footer';
+
+export default function Landing() {
+  useEffect(() => {
+    document.title = 'Landing – BotTrition';
+  });
+
+  return (
+    <div>
+      <h1>Take Charge Of Your Health With BotTrition</h1>
+      <p>
+        Imagine an app that can improve your overall health through lifestyle recommendations.
+        And can not only show detailed food nutritional information through an interactive
+        food search, but also track body physicals to maintain proper health fitness from
+        B.M.I. calculation.
+
+      </p>
+
+      <h3>Motivation</h3>
+      <p>To be able to change and maintain overall health through better choices</p>
+
+      <h3>Why This Matters</h3>
+      <p>
+        Health plays an essential part in how we function as individuals.
+        If proper health is not maintained we could be at risk for life
+        shortening ailments that could hamper our quality of life.
+      </p>
+      <h3>Get started today on your new life today!</h3>
+      <p><a href="/registration">Register</a></p>
+      <p><a href="/login">Login</a></p>
```

Comment:
Since we're using React Router now, internal anchors should be changed to `Link` elements. See #40 for examples of this (`a` → `Link`, `href` → `to`).

Reply:
Actually, I think it may be best to leave these as is and fix them in a future PR. You already have the merge conflict to deal with; these aren't a major issue. I'm going to resolve this.